### DISCUSSION TITLE
How to get the editor's content when submiting a form?

### DISCUSSION THREAD

**Discussion by shyakadavis** [prosekit/prosekit#608] [Q&A]
Hello, 👋 

Does anyone know how to grab the output of the editor for purposes of storing in a dB?

I'm using the `Svelte` bindings, but I believe the same principles apply across others.

Right now, I'm using a work-around of grabbing the value inside the `onDocUpdate` callback, but I would guess there ought to be something simpler such as, `editor.getJSON()`, maybe?

Speaking of, if one is working with PostgreSQL, would you recommend using the `JSONB` format/column type when storing said content? Any other tips you might share in this regard?

Thank you.

**Comment by ocavue** [comment]
https://ocavue-svelte-save-json3--prosekit.netlify.app/examples/save-json could be a good start.
